src/UI/Component/Layout/Page/Factory.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace ILIAS\UI\Component\Layout\Page;
+/**
+ * This is the factory for page-layout components
+ */
+interface Factory {
+
+    /**
+     * ---
+     * description:
+     *   purpose: >
+     *     The Notification Center concentrates the visualization of _any_
+     *     event-notification into one expandable glyph.
+     *     This unification removes the multitude of dedicated notification-icons
+     *     in favor of visual cleanliness as well as providing a designated
+     *     location for further extensions.
+     *
+     *   composition: >
+     *      The Notification Center is visualized as a glyph with a counter.
+     *      Clicked, a pop-over list with the notifying services and their
+     *      respective counter-glyphs is expanded.
+     *
+     *   effect: >
+     *      All notifications, regardless of their origin, are summed up in
+     *      the counter of the Notification Center's glyph.
+     *      When clicked, a list is shown with all notifying services.
+     *      The entries each consist of the services' respective glyph, counter
+     *      and title.
```

Comment:
Pleace move this to the respective entry of this list if one will exist. If not, move to composition.